ethereum/contracts/bridge/BridgeGovernance.sol

```diff
@@ -96,11 +118,11 @@ contract BridgeGovernance is BridgeGetters, BridgeSetters, ERC1967Upgrade {
 
         chain.module = encoded.toBytes32(index);
         index += 32;
-        require(chain.module == module, "invalid RegisterChain: wrong module");
+        require(chain.module == module, "wrong module");
```

Comment:
Had to cut down on some of these strings so circumvent the 24KB contract size limit. Would rather do this than lower the optimizer runs (default: 200).